pkg/util/profiling/settings.go

```diff
@@ -14,7 +14,10 @@ import (
 
 // Settings contains the settings for internal profiling, to be passed to Start().
 type Settings struct {
-	// ProfilingURL specifies the URL to which profiles will be sent.  This can be constructed
+	// Socket specifies the UDS to which profiles will be sent. This takes
+	// precedence over ProfilingURL if set.
```

Comment:
Per above, this new comment isn't correct, but also the existing comments aren't correct.